website/docs/r/nsxt_edgegateway_l2_vpn_tunnel.html.markdown

```diff
@@ -0,0 +1,140 @@
+  provider level. Useful when connected as sysadmin working across different organisations
+* `edge_gateway_id` - (Required) The ID of the edge gateway (NSX-T only). 
+  Can be looked up using `vcd_nsxt_edgegateway` datasource
+* `name` - (Required) The name of the tunnel.
+* `description` - (Optional) The description of the tunnel.
+* `session_mode` - (Required) Mode of the tunnel session (SERVER or CLIENT)
+* `enabled` - (Optional) State of the `SERVER` session, always set to `true` for `CLIENT` 
+  sessions.
+* `connector_initiator_mode` - (Required for `SERVER` sessions) Mode in which 
+  the connection is formed. Only relevant to `SERVER` sessions. One of:
+	* `INITIATOR` - Local endpoint initiates tunnel setup and will also respond to 
+  incoming tunnel setup requests from the peer gateway.
+	* `RESPOND_ONLY` - Local endpoint shall only respond to incoming tunnel setup 
+  requests, it shall not initiate the tunnel setup.
+	* `ON_DEMAND` - In this mode local endpoint will initiate tunnel creation once 
+  first packet matching the policy rule is received, and will also respond to 
+  incoming initiation requests.
+* `local_endpoint_ip` - (Required) The IP address corresponding to the Edge 
+  Gateway the tunnel is being configured on. The IP must be sub-allocated 
+  on the Edge Gateway.
+* `remote_endpoint_ip` - (Required) The IP address of the remote endpoint, which 
+corresponds to the device on the remote site terminating the VPN tunnel.
+* `tunnel_interface` - (Optional) The network CIDR block over which the session 
+  interfaces. Relevant only for SERVER session modes. If not provided, Cloud 
+  Director will attempt to automatically allocate a tunnel interface.
+* `pre_shared_key` - (Required for `SERVER` sessions) The key that is used for 
+  authenticating the connection, only needed for `SERVER` sessions.
+* `peer_code` - (Optional) Encoded string that contains the whole configuration 
+  of a `SERVER` session, including the pre-shared key, so it is user's 
+  responsibility to secure it.
+* `stretched_network` - (Optional) One or more stretched networks for the tunnel. 
+  See [`stretched_network`](#stretched-network) for more detail.
+
+## Stretched network
+
+* `network_id` - (Required) Network ID of a routed network on the Edge Gateway. 
+  Can be looked up using [`vcd_routed_network_v2`](/providers/vmware/vcd/latest/docs/resources/nsxt_edgegateway_l2_vpn_tunnel) 
+  datasource.
+* `tunnel_id` - (Optional) Tunnel ID of the network on the tunnel, required for 
+  `CLIENT` sessions, computed for `SERVER` sessions.
+
+
+## Importing
+
+~> The current implementation of Terraform import can only import resources into the state.
+It does not generate configuration. [More information.](https://www.terraform.io/docs/import/)
+
+An existing L2 VPN Tunnel configuration can be [imported][docs-import] into this resource
+via supplying path for it. An example is
+below:
```

Comment:
Nit: this line could be part of the line above.

Reply:
Updated